go/api/openapi.yaml

```diff
@@ -57,6 +57,30 @@ paths:
           description: >-
             Successfully created a Personal Access Token
 
+  /token:
+    post:
+      summary: Refresh the JWT access token
+      tags:
+        - token
+        - access-token
+        - jwt
+      security:
+        - BearerAuthElevated: []
```

Comment:
this doesn't need the elevated security scheme, just the `BearerAuth`

Reply:
true, copy/paste issue